4.8/alpine/Dockerfile

```diff
@@ -48,20 +48,22 @@ RUN addgroup -g 1000 node \
 
 ENV YARN_VERSION 0.24.4
 
-RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg \
+RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
```

Comment:
@Daniel15 it only has the stripped-down `tar` version provided by BusyBox by default, which does not support the `--strip-components` option. So we temporarily install the "real" `tar` to make life easier.